Formula/coq.rb

```diff
@@ -48,4 +48,10 @@ def caveats; <<-EOS.undent
       (autoload 'coq-mode "coq" "Major mode for editing Coq vernacular." t)
     EOS
   end
+
+  test do
+    unless File.exist? "#{bin}/coqc" then return false end
+    system("#{bin}/coqc", "-v")
+    $?.exitstatus == 0
```

Comment:
Also if `coqc` doesn't exist and you call it in the test it fails with a pretty obvious error anyway, we don't need to throw a manual error ourselves there.